randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -177,17 +177,35 @@ Extra - asset_id: 4
   * Normal Door to The Moat/Door to West Ocean
   > Door to Wrecked Ship Entrance
       Any of the following:
-          # TODO: Check to see if possible with HBJ.
           Grapple Beam or Space Jump
-          # It is possible to jump across by not using full jumps to avoid bonking on the ceiling.
+          Horizontal Infinite Bomb Jump (Expert) and Can Lay Bombs
+          # It is possible to jump across by not using full jumps to avoid bonking on the celing.
           Movement (Intermediate)
           All of the following:
-              # From the sea floor
+              # From the Sea Floor
               Gravity Suit
               Any of the following:
                   Hi-Jump Boots or Wall Jumps (Beginner)
                   # IBJ
                   Vertical Infinite Bomb Jump (Beginner) and Can Lay Bombs
+  > Door from Gravity Suit Room
+      Any of the following:
+          Space Jump
+          All of the following:
+              # Get over and WJ up
+              Grapple Beam or Movement (Intermediate)
```

Comment:
just a thought, is movement still needed with the HJ? AFAIK you can cross the gaps with just HJ but i might be wrong.

Reply:
Its actually harder to get over the gaps with HJ on. The movement is there because you need to do lower more precise jumps to make the gap.